roofit/roofit/src/RooArgusBG.cxx

```diff
@@ -111,8 +111,16 @@ Double_t RooArgusBG::analyticalIntegral(Int_t code, const char* rangeName) const
   Double_t f1 = (1.-TMath::Power(min/m0,2));
   Double_t f2 = (1.-TMath::Power(max/m0,2));
   Double_t aLow, aHigh ;
-  aLow  = -0.5*m0*m0*(exp(c*f1)*sqrt(f1)/c + 0.5/TMath::Power(-c,1.5)*sqrt(pi)*RooMath::erf(sqrt(-c*f1)));
-  aHigh = -0.5*m0*m0*(exp(c*f2)*sqrt(f2)/c + 0.5/TMath::Power(-c,1.5)*sqrt(pi)*RooMath::erf(sqrt(-c*f2)));
+  if ( c < 0. ) { 
+    aLow  = -0.5*m0*m0*(exp(c*f1)*sqrt(f1)/c + 0.5/TMath::Power(-c,1.5)*sqrt(pi)*RooMath::erf(sqrt(-c*f1)));
+    aHigh = -0.5*m0*m0*(exp(c*f2)*sqrt(f2)/c + 0.5/TMath::Power(-c,1.5)*sqrt(pi)*RooMath::erf(sqrt(-c*f2)));
+  } else if ( c == 0. ) {
+    aLow  = -m0*m0/3.*f1*sqrt(f1);
+    aHigh = -m0*m0/3.*f1*sqrt(f2);
+  } else {
+    aLow  = 0.5*m0*m0*exp(c*f1)/(c*sqrt(c)) * (0.5*sqrt(pi)*((std::complex<double>)RooMath::faddeeva(sqrt(c*f1))).imag() - sqrt(c*f1));
```

Comment:
Can you remove the (std::complex<double>) casts? They seem to be unnecessary because the function returns just that.

Reply:
Yes, I'll remove them.

Reply:
Unnecessary casts are removed and committed and pushed.